.editorconfig

```diff
@@ -37,4 +37,6 @@ dotnet_sort_system_directives_first = true
 # Expression-level preferences
 csharp_style_implicit_object_creation_when_type_is_apparent = true:warning
 # IDE0005: Using directive is unnecessary.
-dotnet_diagnostic.IDE0005.severity = suggestion
\ No newline at end of file
+dotnet_diagnostic.IDE0005.severity = suggestion
+# IDE0290: [Use primary constructor](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0290)
+dotnet_diagnostic.IDE0290.severity = suggestion
```

Comment:
This isn't a convention we currently follow, and I think it warrants a discussion with the team before enforcing it.

I personally find this less readable, and don't see the value in enforcing it, but would be happy to hear opinions from others.